restclient-jq.el

```diff
@@ -6,7 +6,8 @@
 ;; Author: Cameron Dorrat <cdorrat@gmail.com>
 ;; Maintainer: Cameron Dorrat <cdorrat@gmail.com>
 ;; Created: 26 Apr 2020
-;; Keywords: http jq
+;; Keywords: tools comm http jq
+;; Version: 0.1
```

Comment:
package-lint throws a warning when `Version:` is missing. I see that `restclient.el` doesn't have semantic versioning, but I don't know whether there's any reasoning behind it. Also, adopting semantic versioning for one single component but not for the whole project might not make a lot of sense. But then, maybe there's no reasoning and I'm overthinking this issue. ;-)